deploy/packages/bbp-packages.yaml

```diff
@@ -55,6 +55,7 @@ packages:
       - touchdetector@4.4.2
       - touchdetector@5.0.1
       - touchdetector@5.1.0
+      - brion@3.0.0
```

Comment:
Please also whitelist `brion` in `deploy/configs/applications/modules.yaml`

Reply:
From our [README](https://github.com/BlueBrain/spack#managing-centrally-built-modules-and-packages):
> To actually enable the generation of corresponding module files, new software has to be whitelisted in the module configuration for spack, e.g. by editing:
>
>     $ vim spack/deploy/configs/applications/modules.yaml
>
> Look for a key whitelist, and add the package. Packages listed in the module whitelist should be as generic as possible, i.e., not tied to specific versions.

Reply:
Fell into the same trap again :) sorry, and thanks for reminding about the updated readme!